charts/rancher-webhook/105.0.1+up0.7.0-rc.3/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: < 1.32.0-0
+  catalog.cattle.io/managed: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.10.0-0 < 2.11.0-0'
+  catalog.cattle.io/release-name: rancher-webhook
+apiVersion: v2
+appVersion: 0.7.0-rc.3
+description: ValidatingAdmissionWebhook for Rancher types
+name: rancher-webhook
+version: 105.0.1+up0.7.0-rc.3
```

Comment:
Seems like we've missed something in the automation. That version should be 

`106.0.0`, not `105.0.1` because `105` is for 2.10